### DISCUSSION TITLE
About use adt to packaging SWF game by cmd

### DISCUSSION THREAD

**Discussion by TimeRunis** [airsdk/Adobe-Runtime-Support#2528] [Q&A]
I use the adt.jar of SDK to package an SWF game by cmd. But it will become black background after the Harman logo. In front of the black background, the game UI and text could still be seen. Due to the game needing an element of the original background to start, so I can't start the game.

**Comment by ajwfrost** [comment]
Hi

Which platform is this on? and how are you displaying the original background?
Please also confirm which AIR SDK you're using?
If you're using Stage3D, do you see any errors if you enable the error checking?

thanks

**Reply by TimeRunis** [reply]
This is the content of the BAT file that i use.

> "D:\Softwares\Java\Java16\bin\java.exe" -Xmx2048M -jar "H:\sdk\AIRSDK_Windows\lib\adt.jar" -package -target apk -storetype pkcs12 -keystore "H:\sdk\2333.pfx" "H:\swf\2333.apk" "H:\swf\application.xml" "H:\swf\start.swf" "H:\swf\lib"

i can't check the log or any error in this way, so i don't know any information. And I using AIR 50.2 SDK.
i have a try `<application xmlns="http://ns.adobe.com/air/application/32.0">`  and `<application xmlns="http://ns.adobe.com/air/application/50.0">` in the `application.xml`.But don't work

**Reply by ajwfrost** [reply]
Could you send us the APK file so that we can try this here? If you could package it up with "apk-debug" instead of "apk" it would help..
https://transfer.harman.com/requests/l28Mi2Tpibz5qDhuhvDMqt

thanks
